frontend/app/(drawer)/profile.tsx

```diff
@@ -7,270 +7,198 @@ import {
+      //   body: JSON.stringify({
+      //     subject: 'Problem Report from Idioms App',
+      //     message: `User reported a problem with the app.\n\nDevice Info:\n- Platform: ${deviceInfo.platform}\n- App Version: ${deviceInfo.appVersion}\n- Time: ${deviceInfo.timestamp}`,
+      //     userEmail: 'user-report@idioms-app.com', // or get from user if available
+      //   }),
+      // });
+
+      // For now, show success message
+      console.log('Problem report sent:', {
+        platform: Platform.OS,
+        appVersion: '1.0.0',
+        timestamp: new Date().toISOString(),
+      });
 
-  const handleSendFeedback = useCallback(() => {
-    if (!feedbackForm.subject.trim() || !feedbackForm.description.trim()) {
       Alert.alert(
-        'Required Fields',
-        'Please complete the subject and description.',
+        'Report Sent!',
+        'Thank you for reporting the issue. Our support team will review it and work on a solution.',
+        [{ text: 'OK' }],
+      );
+    } catch (error) {
+      Alert.alert(
+        'Cannot Send Report',
+        'Please try again later or contact us directly at support@idioms-app.com',
         [{ text: 'OK' }],
       );
-      return;
     }
-    setShowFeedbackModal(false);
-    setFeedbackForm({
-      subject: '',
-      description: '',
-      email: '',
-    });
-    showSuccessNotification('Feedback sent! Thank you.');
-  }, [feedbackForm, showSuccessNotification]);
-
-  const handleSubjectChange = useCallback((text: string) => {
-    setFeedbackForm((prev) => ({ ...prev, subject: text }));
-  }, []);
-
-  const handleDescriptionChange = useCallback((text: string) => {
-    setFeedbackForm((prev) => ({ ...prev, description: text }));
-  }, []);
-
-  const handleEmailChange = useCallback((text: string) => {
-    setFeedbackForm((prev) => ({ ...prev, email: text }));
-  }, []);
+  };
 
-  const resetFeedbackForm = useCallback(() => {
-    setFeedbackForm({
-      subject: '',
-      description: '',
-      email: '',
-    });
-    setShowFeedbackModal(false);
-  }, []);
+  const handleReportProblem = () => {
+    Alert.alert(
+      'Report a Problem',
+      "This will send a problem report to our support team. We'll investigate and work on fixing any issues.",
+      [
+        { text: 'Cancel', style: 'cancel' },
+        {
+          text: 'Send Report',
+          onPress: sendProblemReport,
+        },
+      ],
+    );
+  };
 
   const SettingItem = ({
     title,
     description,
     onPress,
     rightComponent,
+    last = false,
   }: {
     title: string;
-    description: string;
+    description?: string;
     onPress?: () => void;
     rightComponent?: React.ReactNode;
+    last?: boolean;
   }) => (
     <TouchableOpacity
       onPress={onPress}
       style={{
         backgroundColor: colors.surface,
-        borderColor: colors.border,
+        paddingVertical: 16,
+        paddingHorizontal: 20,
+        borderBottomWidth: last ? 0 : 1,
```

Comment:
Why do we have all this code changing? Shouldn't this PR be only about the authentication?